tests/lib/prepare-project.sh

```diff
@@ -32,6 +44,28 @@ build_deb(){
     cp ../*.deb "$GOHOME"
 }
 
+fedora_build_rpm() {
```

Comment:
I have a more generic `build_rpm` now. Lets land this as is and I will propose a PR once the suse/fedora ones landed to convert both into a single `build_rpm` function. @niemeyer @Conan-Kudo OK for you guys?